minio/minio
# Replaces 'disk'=>'drive' visible to end user

## Description
Replaces the word `"Disk"` with `"Drive"` per AB's comment:
>  we should find and replace Disk with Drive everywhere. Disk is HDD. Drive is neutral for all HDDs and SSDs.

## Motivation and Context
Disk is HDD. Drive is neutral for all HDDs and SSDs.

## How to test this PR?
Run existing tests and make sure there is no references to `"Drive"`s as `"Disk"`s.

## Types of changes

## Checklist:
